Functions.cmake

```diff
@@ -87,3 +87,17 @@ function(install_webplugin _dir)
   file(GLOB _userguides "${_dir}/userguide/*.md")
   set_property(GLOBAL APPEND PROPERTY USERGUIDES "${_userguides}")
 endfunction(install_webplugin)
+
+# Finds the absolute paths for the given Boost libraries
+# Use variable arguments for the Boost libraries to link
+function(find_boost_library)
+  foreach(_lib ${ARGV})
+    foreach(_path ${Boost_LIBRARIES})
+      if(_path MATCHES ".*${_lib}\.so$")
```

Comment:
Because every Boost library seemingly has a `boost_` prefix, this could be hardcoded here.

Reply:
Updated accordingly.